frontend/src/pages/profile/profile.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from '~/libs/components/components.js';
+import { useAppSelector } from '~/libs/hooks/hooks.js';
+
+import avatar from '../../assets/img/avatar.jpg';
+import styles from './styles.module.scss';
+
+const Profile: React.FC = () => {
+  const { user } = useAppSelector(({ auth }) => ({
+    user: auth.user,
+  }));
+
+  return (
+    <div className={styles.profile}>
+      <div className={styles.user_info_block}>
+        <div className={styles.user_avatar}>
+          <img className={styles.user_avatar_image} src={avatar} alt="avatar" />
```

Comment:
Check comment about avatar placeholder approach
https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/28#discussion_r1301103808

Reply:
Should I create a separate common component for an avatar now, or is it better to do it in a separate task? 
For now, I've changed it like this

  const avatarUrl = null;
  
          {avatarUrl ? (
            <img
              className={styles.userAvatarImage}
              src={avatarUrl}
              alt="avatar"
            />
          ) : (
            <div className={styles.defaultAvatar}>
              <span className={styles.initials}>{initials}</span>
            </div>
          )}

Reply:
You may wait for Avatar changes with it